O/osslsigncode/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "osslsigncode"
+version = v"2.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mtrojnar/osslsigncode",
+                  "4568c890cc1538ca80be3ee36775ba42223dea04"),
+]
+
+script = raw"""
+    cd $WORKSPACE/srcdir/osslsigncode
+
+    sed -i 's/Ws2_32/ws2_32/g' CMakeLists.txt
+
+    cmake -B build \
+        -DCMAKE_INSTALL_PREFIX=${prefix} \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DZLIB_INCLUDE_DIR=/workspace/destdir/include \
+        -DZLIB_LIBRARY=/workspace/destdir/lib/libz.a
```

Comment:
I'm confused: if you're linking the static library of zlib, why is that a runtime dependency?  Why not using the shared library instead?

Reply:
Simply me being risk-averse, I guess. I changed to a shared library instead.